frontend/views/DetailsPanel.js

```diff
@@ -144,6 +148,15 @@ module.exports = kind({
     // This is data loaded from the web service
     {from: 'packageInfo.title', to: 'title'},
     {from: 'packageInfo.id', to: 'titleBelow'},
+    {
+      from: 'packageInfo.appDescription', to: '$.appDescription.content', transform: function (description) {
+        var appDescription = 'No description provided for this package'
+        if (description instanceof String && description.trim().length) {
```

Comment:
Out of curiosity - have you actually tested this yet? ;)

```
>>> "abc" instanceof String
 <- false
```

Reply:
You are right. instanceof only works with new.
Correcting it.